.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  extends: '@ethereumjs/eslint-config-defaults',
+  parserOptions: {
+    project: ['./tsconfig.json']
+  },
+  rules: {
+    '@typescript-eslint/no-floating-promises': 'off',
+    'no-redeclare': 'off'
```

Comment:
I turned this off for now as the only problem here was duplication of names in the "export namespace" for ETH, LES.

It seems like we can solve this with setting `ignoreDeclarationMerge` to true, but that option seems to only be available in the rule for `typescript-eslint/no-redeclare` where here we are using the base `no-redeclare` and I didn't want to add the typescript-eslint module just to add that rule. The config would look something like this:

```typescript
{
  // note you must disable the base rule as it can report incorrect errors
  "no-redeclare": "off",
  "@typescript-eslint/no-redeclare": ["error", { ignoreDeclarationMerge: true } ]
}
```

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-redeclare.md